src/FileServiceManager/FileService/DAL/FileSystemConfiguration/FileSystemJsonConfiguration/FileSystemJsonConfigurator.cs

```diff
@@ -0,0 +1,229 @@
+﻿using FileService.Models;
+using Microsoft.Extensions.Configuration;
+using Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog.Fluent;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+
+/// <summary>
+/// FileSystemJsonConfigurator implementing IFileSystemConfigurator. Used to read/write/remove json configuration about company's file system structure
+/// </summary>
+
+namespace FileService.DAL
+{
+    public class FileSystemJsonConfigurator : IFileSystemConfigurator
+    {
+        private readonly string _configPath = "";
+
+        public FileSystemJsonConfigurator(IConfiguration configuration)
```

Comment:
pass to this class IOptions of the configuration that it needs, extract the IConfiguration injection from class itself

Reply:
Done